initializr-service/src/main/resources/application.yml

```diff
@@ -1253,6 +1261,38 @@ initializr:
           versionRange: "[1.3.0.RELEASE,2.0.0.M1)"
           groupId: org.springframework.boot
           artifactId: spring-boot-actuator-docs
+    - name: Spring Cloud GCP
+      bom: spring-cloud-gcp
+      versionRange: 2.0.0.RC2
+      content:
+        - name: GCP Support
+          id: cloud-gcp
+          description: Spring Boot support to use GCP services through Spring Cloud
```

Comment:
Everything here is about Spring Boot so I am not sure this adds a lot of value. A proper description is very important. I'd rather focus on what that allows you to do. You can be very generic (Google Cloud Platform support) or list the services that it integrates.